lib/peg.d.ts

```diff
@@ -0,0 +1,137 @@
+// Based on PEG.js Type Definitions by: vvakame <https://github.com/vvakame>, Tobias Kahlert <https://github.com/SrTobi>, C.J. Bell <https://github.com/siegebell>
+
+declare namespace PEG {
+  function parse(input: string): any;
+
+  interface Location {
+    line: number;
+    column: number;
+    offset: number;
+  }
+
+  interface LocationRange {
+    start: Location;
+    end: Location;
+  }
+
+  class SyntaxError {
+    line: number;
+    column: number;
+    offset: number;
+    location: LocationRange;
+    expected: any[];
+    found: any;
+    name: string;
+    message: string;
+  }
+}
+
+export type Location = PEG.Location;
+export type LocationRange = PEG.LocationRange;
+
+export interface ExpectedItem {
+  type: string;
+  value?: string;
+  description: string;
+}
+
+export interface PeggyError extends Error {
+  name: string;
+  message: string;
+  location: LocationRange;
+  found?: any;
+  expected?: ExpectedItem[];
+  stack?: any;
+}
+
+// for backwards compatibility with PEGjs
+export interface PegjsError extends PeggyError {}
+
+export type GrammarError = PeggyError;
+export var GrammarError: any;
+
+export interface ParserOptions {
+  startRule?: string;
+  tracer?: any;
```

Comment:
`tracer` is almost undocumented. We can leave fixing this as a TODO, but let's file a post-1.0 bug.

Reply:
See #24